.gitignore

```diff
@@ -77,6 +77,9 @@ jspm_packages/
 # OS X temporary files
 .DS_Store
 
+# IntelliJ Platform files
+.idea/
+
```

Comment:
Maybe we can also add .vscode

Reply:
Not always, we're using recommended extensions there and some settings specific to certain extensions (graphql)